postgresql/cluster/bdr/master.sls

```diff
@@ -0,0 +1,33 @@
+{%- from "postgresql/map.jinja" import server with context %}
+{%- from "postgresql/map.jinja" import cluster with context %}
+{%- if cluster.enabled %}
+
+{%- set repuser = cluster.user.get('repuser') %}
```

Comment:
Yes I don't see reason to parametrize this, the user will always be there (replication won't work otherwise), I just parametrized this users password